src/llmq/quorums_instantsend.cpp

```diff
@@ -410,6 +410,23 @@ bool CInstantSendManager::ProcessTx(const CTransaction& tx, bool allowReSigning,
         return false;
     }
 
+    // Only sign for inlocks or islocks if IS Signing is enabled
+    // However, if we are signing a tx, b/c it was included in a block we should
+    // sign even if spork disabled. This allows a ChainLock to happen on this
+    // block after we retroactively islock
```

Comment:
```suggestion
    // Only sign for inlocks or islocks if mempool IS signing is enabled.
    // However, if we are processing a tx because it was included in a block we should
    // sign even if mempool IS signing is disabled. This allows a ChainLock to happen on this
    // block after we retroactively locked all transactions.
```